lib/serializer.js

```diff
@@ -58,7 +58,15 @@ module.exports = function serializer(msg){
 		});
 	}
 	function attr(group, name, obj){
-		var syntax = attributes[group][name];
+		var syntax;
+
+    if (Array.isArray(group)) {
+			group.forEach((grp) => {
+			  syntax = syntax || attributes[grp][name];
+			});
+		} else {
+			syntax = attributes[group][name];
+		}
```

Comment:
Let's go this route:
```
	var syntax = Array.isArray(group)
		? group.find((grp) => { return attributes[grp][name] })
		: attributes[group][name];
```
Array.find has been available for a long time. As long as the ability to use arrow functions ;) Node will stop supporting the older versions without it soon anyhow.